main/EDTypesMod.F90

```diff
@@ -31,6 +31,8 @@ module EDTypesMod
   use EDParamsMod,           only : nclmax, nlevleaf, maxpft
   use FatesConstantsMod,     only : n_dbh_bins, n_dist_types
   use shr_log_mod,           only : errMsg => shr_log_errMsg
+  use SFFireWeatherMod,      only : fire_weather
+  use SFNesterovMod,         only : nesterov_index
```

Comment:
This can be removed since `fireWeather` is being defined using the base abstract class, correct?
```suggestion
```

Reply:
oh hmm maybe, I thought I tried this and it wouldn't compile but I will try again.